### DISCUSSION TITLE
Start version for unterschiedliche Repository-Pfade

### DISCUSSION THREAD

**Discussion by MHofmann-rossmanith** [PaulHatch/semantic-version#130] [Q&A] [answered]
Hi,

I want to set a different starting version for 5 paths in my repo.
How can I implement this?

The tag variant only applies to the entire repository.

**Comment by PaulHatch** [comment, marked as answer]
You want to have a path in the repository that uses a different version without impacting the primary repo version? You would add an additional version step in this case, configured to look at the path and use a namespace and change_path inputs. Take a look at the [namespace section in the configuration guide](https://github.com/PaulHatch/semantic-version/blob/master/guide.md#namespace-services--monorepo-support).

Most likely this would go in an additional job, for example to build the specific service located in that subfolder.
